### PR TITLE
Added option to slide from left to right

### PR DESCRIPTION
**Pull request by anevins12**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,6 +8,16 @@
         <link rel="stylesheet" media="screen" href="gh-pages/css/styles.css" />
         <link rel="stylesheet" media="screen" href="gh-pages/css/demo.css" />
         <link rel="stylesheet" media="screen" href="gh-pages/css/rainbow/github.css" />
+        <style>
```
Comment (lawlesscreation) [inline index.html]: These styles should go in gh-pages/css/demo.css

File: README.md [hunk 2]
```diff
@@ -121,6 +121,7 @@ Description: The text for the button that shows the content
 Type: `string` or `number`
 
 Default: 'slow'
+Other values: 'none', '0' or 0
```
Comment (lawlesscreation) [inline README.md]: There are more values available than this. Either list all values or explain what 'none', '0' or 0 are different

File: index.html [hunk 3]
```diff
@@ -86,6 +96,34 @@ <h3 id="triggerC">Contact details</h3>
...
 </pre>
 
+            <h2>Example E</h2>
+            <p>Custom CSS animation.</p>
+            <div class="example exampleG">
+                <p>Closes on click elsewhere in the document.</p>
+            </div><!-- .exampleC -->
+
+            <h3>Initialisation</h3>
+<h4>CSS</h4>
```
Comment (lawlesscreation) [inline index.html]: Picky point but this should be indented like line 105. Only <pre> elements need their white space removed.

**Comment by anevins12** [comment]
I've made those changes & merged in bug-16 branch

**Pull request closed without merging**